portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,85 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
```

Comment:
Where is this file?

Reply:
The files are:
script.js: https://github.com/tzavidas/step-tutorial/blob/master/portfolio/src/main/webapp/script.js
style.css: https://github.com/tzavidas/step-tutorial/blob/master/portfolio/src/main/webapp/style.css